x-pack/plugins/security_solution/public/explore/network/components/embeddables/embedded_map.test.tsx

```diff
@@ -5,221 +5,231 @@
  * 2.0.
  */
 
-import type { ReactWrapper } from 'enzyme';
-import { mount, shallow } from 'enzyme';
+import { render, waitFor } from '@testing-library/react';
```

Comment:
To make these easier to contribute to, I refactored these tests to use react testing library over enzyme